docs/semgrep-appsec-platform/notifications.md

```diff
@@ -4,75 +4,16 @@ append_help_link: true
-
-## Finding available alert and notification channels
-
-To find available [integrations for Semgrep AppSec Platform](https://semgrep.dev/orgs/-/settings/integrations), follow these steps:
-
-1. Sign in to your [Semgrep AppSec Platform account](https://semgrep.dev/).
-2. Click **Settings**.
-3. Click **Integrations**.
-    ![Screenshot of Semgrep's "Create New Integration Channel" menu](/img/integration-firstview.png)
-4. Click **Add Integration** (or **Setup First Integration** if this is your first integration).
-    ![Screenshot of Integrations page while adding the first integration.](/img/integrations.png)<br />
-
-## Managing alerts and notifications
-
-To view, add, remove, disable, or enable your saved channels:
-
-1. In the **Settings** > **[Integrations](https://semgrep.dev/orgs/-/settings/integrations)** page, explore the options available for specific integrations.
-2. In the **[Policies](https://semgrep.dev/orgs/-/policies)** page, click **Rule modes**. A menu appears.
-3. Click the **Edit** button of the mode for which mode you want to change notifications.
-4. Make any changes to the notification settings for the mode you selected.
-
-### Semgrep Autofix
-
-[Autofix](/writing-rules/autofix) is a Semgrep feature in which rules contain suggested fixes to resolve findings. Either metavariables or regex matches are replaced with a potential fix. Due to their complexity, not all rules make use of autofix, but for rules that use this feature, autofix allows you to quickly resolve findings as part of your code review workflow. Semgrep Code can suggest these fixes through PR or MR comments within GitHub or GitLab, thus integrating seamlessly with your review environment.
-
-Autofix is free to use for all tiers.
-
-In the following screenshot, Semgrep detects the use of a native Python XML library, which is vulnerable to XML external entity (XXE) attacks. The PR comment automatically suggests a fix by replacing `import xml` to `import defusedxml`.
-
-![Screenshot of a sample autofix PR suggestion](/img/notifications-github-suggestions.png)
-
-#### Enabling autofix for GitHub or GitLab
-
-Autofix requires PR or MR comments to be enabled for your repository or organization. Follow the steps in [GitHub pull request comments](/semgrep-appsec-platform/github-pr-comments) or [GitLab merge request comments](/semgrep-appsec-platform/gitlab-mr-comments) to enable this feature.
-
-<EnableAutofix />
-
-All scans performed after enabling autofix generate inline PR or MR comments with code suggestions for applicable rules.
-
-## Notification and alert de-duplication
-
-Notifications are sent only the first time a given finding is detected.
-
-Because of Semgrep CI's diff-awareness, you will not be notified
-when a pull request has a finding that existed on the base branch already,
-even if that line is moved or re-indented.
+Users with team or enterprise subscriptions can receive notifications from Semgrep AppSec Platform in the following channels:
```

Comment:
```suggestion
Users can receive notifications from Semgrep AppSec Platform in the following channels:
```
(I think we can remove those bits because our docs have shifted towards "commercial by default".)